pallets/corporate-actions/src/distribution/mod.rs

```diff
@@ -183,6 +184,13 @@ decl_module! {
         /// The distribution will commence at `payment_at` and expire at `expires_at`,
         /// if provided, or if `None`, then there's no expiry.
         ///
+        /// However, the funds will be locked in `portfolio` when `distribute` is called,
+        /// and not when `payment_at` is due.
+        /// So when there's no expiry, some funds may be locked indefinitely in `portfolio`,
+        /// due to claimants not withdrawing.
+        /// This cannot as a result of dust from rounding down indivisible currency amounts,
```

Comment:
```suggestion
        /// This can also happen as a result of dust from rounding down indivisible currency amounts,
```

Reply:
It cannot though. We do `Self::unlock(&dist, benefit)?;`, not `Self::unlock(&dist, gain)?;`, and any fraction below `ONE_UNIT`, that gets cut off from `gain`, will still be in `benefit`.

Reply:
hmm, shouldn't the round down be done for the benefit before the tax calculation?

Reply:
I also wondered about this. In theory a gain could get rounded down to zero and a withholding tax would still be registered for an individual. For transfers of benefits only gain matters, so it makes sense that if using a indivisible currency that it is rounded down, but the recorded withholding tax does end up slightly off based on the actual gain distributed. Unless the currency unit is of significant value I'm not sure if it would be significant from an accounting perspective. 

Using an indivisible currency is not really ideal for capital distribution. I'd previously given feedback as part of my experience with the ITN that I felt capital distribution should be restricted to only divisible currencies because of the above.

Reply:
Usually though any withholding tax will be lower %-wise than what you get to keep, so if gains are rounded then taxes would be as well when the agent actually goes to pay them.

> Unless the currency unit is of significant value I'm not sure if it would be significant from an accounting perspective.

Yeah I think that's an important point. Tax authorities don't really care about rounding errors. There's no incentive for them to go after that.

> I'd previously given feedback as part of my experience with the ITN that I felt capital distribution should be restricted to only divisible currencies because of the above.

Might be a good direction, but I think for now we decided to round down.

In closing I think rounding down before gain would require rounding down after as well due to taxes, so its a more complicated procedure overall which can leave more dust. So let's stick with rounding down `gain` only.